onnx/reference/custom_element_types.py

```diff
@@ -5,16 +5,27 @@
 
 import warnings
 
-from onnx.custom_element_types import bfloat16  # noqa: F401
-from onnx.custom_element_types import float8e4m3fn  # noqa: F401
-from onnx.custom_element_types import float8e4m3fnuz  # noqa: F401
-from onnx.custom_element_types import float8e5m2  # noqa: F401
-from onnx.custom_element_types import float8e5m2fnuz  # noqa: F401
-from onnx.custom_element_types import int4  # noqa: F401
-from onnx.custom_element_types import uint4  # noqa: F401
+from onnx.custom_element_types import bfloat16
+from onnx.custom_element_types import float8e4m3fn
+from onnx.custom_element_types import float8e4m3fnuz
+from onnx.custom_element_types import float8e5m2
+from onnx.custom_element_types import float8e5m2fnuz
+from onnx.custom_element_types import int4
+from onnx.custom_element_types import uint4
 
 warnings.warn(
     "This file was moved to onnx.custom_element_types. Importing this module will fail in onnx>=1.19.0.",
-    category=DeprecationWarning,
+    category=PendingDeprecationWarning,
     stacklevel=1,
 )
+
+
+__all__ = [
```

Comment:
nit: All should be placed before the imports: https://peps.python.org/pep-0008/#module-level-dunder-names

> Module level “dunders” (i.e. names with two leading and two trailing underscores) such as `__all__`, `__author__`, `__version__`, etc. should be placed after the module docstring but before any import statements except from `__future__` imports. Python mandates that future-imports must appear in the module before any other code except docstrings:

Reply:
removed